recipes/hazelbean/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "hazelbean" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hazelbean-{{ version }}.tar.gz
+  sha256: 43cda7cf51ed5a57b3bf32665f572ff245e84a33d977e85fcc42eb0e13b99466
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - python >=3.6
+    - pip
+    - cython <3
+    - numpy
+    - setuptools
+    - setuptools_scm
+    - {{ stdlib("c") }}
+  run:
+    - Cython
+    - Pyro4
+    - aenum
+    - anytree
+    - dask
+    - descartes
+    - fiona
+    - geopandas
+    - google-cloud-datastore
+    - google-cloud-storage
+    - libgcc
+    - markdown
+    - matplotlib-base
+    - natcap.invest
+    - netcdf4
+    - numpy
+    - openpyxl
+    - gdal
+    - pandas
+    - pandoc
+    - pygeoprocessing
+    - pyqt
+    - python >=3.6
+    - pyyaml
+    - qtawesome
+    - qtpy
+    - rioxarray
+    - scikit-learn
+    - scipy
+    - seaborn
+    - setuptools
```

Comment:
You rarely need setuptools at run time. Are you using `pkg_resources`?

```suggestion
```

Reply:
You're right, `setuptools` is only needed at build time, not at runtime.